### ISSUE TITLE
Change Base Image (?)

### ISSUE THREAD

**Issue by t-book** [GeoNode/geoserver-docker#69]
Due to security it would be advisable to switch to one of these base images:

```
Base Image                             Vulnerabilities  Severity
tomcat:10.0.0-M7-jdk14-openjdk-oracle  0                0 high, 0 medium, 0 low
tomcat:9.0.37-jdk14-openjdk-oracle     0                0 high, 0 medium, 0 low
tomcat:10.0.0-M5-jdk14-openjdk-oracle  0                0 high, 0 medium, 0 low
tomcat:9.0.31-jdk13-openjdk-oracle     0                0 high, 0 medium, 0 low
```

As from what I see these are all on centos we should check for a newer working buster image.

**Comment by frafra** [comment]
I agree with updating the images; we could enable depend-a-bot (now integrated in GitHub) to check for security upgrades for the Dockerfiles.
I have some doubts in moving from the tomcat image to something custom based on Debian if that is because of CentOS Stream, but it could worth the effort if that could help to streamline the non-Docker setup with the Dockerfiles.

**Comment by t-book** [comment]
> I agree with updating the images; we could enable depend-a-bot (now integrated in GitHub) to check for security upgrades for the Dockerfiles.

my +1 and maybe on merging a build push to docker hub as well.

> I have some doubts in moving from the tomcat image to something custom based on Debian if that is because of CentOS Stream, but it could worth the effort if that could help to streamline the non-Docker setup with the Dockerfiles.

I think this is a misunderstanding. From a quick look, I've guessed the current image is Debian based were mentioned above are centos. To not produce too much work with reworking the dockerfile my idea was to stick with a Debian based.

**Comment by frafra** [comment]
> I think this is a misunderstanding. From a quick look, I've guessed the current image is Debian based were mentioned above are centos. To not produce too much work with reworking the dockerfile my idea was to stick with a Debian based.

You are right. It is perfect then :)

**Comment by t-book** [comment]
@frafra do you have experience with what is needed to let depandabot scan Dockerfiles?

**Comment by frafra** [comment]
@t-book dependabot needs to know the location of the Dockerfile and it assumes it is on the main directory by default: https://dependabot.com/docs/config-file/
